SMLHelper/Assets/PrefabInfo.cs

```diff
@@ -8,11 +8,12 @@ namespace SMLHelper.Assets;
 using Handlers;
 
 /// <summary>
-/// Data class that represents information about a prefab.
+/// <para>Data class that represents information about a prefab.</para>
+/// <para>Also see <see cref="WithTechType"/>, which may be more convenient than directly calling the constructor.</para>
```

Comment:
Instead of a paragraph at the top, I think it would be better if we use a `seealso` tag. This will help make it better looking in our [API docs](https://subnauticamodding.github.io/SMLHelper/api/SMLHelper.Assets.html) as well.

I included an existing example below.

https://github.com/SubnauticaModding/SMLHelper/blob/38556d8972ca0b8d99f90abe9955668956c1f699/SMLHelper/Handlers/ConsoleCommandsHandler.cs#L54-L56